MC/dpgsim.sh

```diff
@@ -536,6 +543,20 @@ export CONFIG_REMOVETRACKREFS
 
 # <<<------------------ decide if TrackRefs.root should be removed -----------------<<<
 
+# >>>------------------ decide if signal should be filtered (for embedding) -------->>>
+
+# check consistency of provided (if any) fraction of subjobs where signal filtering is used
+if [[ ! $CONFIG_SIGNALFILTERINGFRACTION =~ ^[0-9]+$ ]] ; then
+    echo "Invalid value $CONFIG_SIGNALFILTERINGFRACTION provided for signalFilteringFraction"
+    exit 1
+fi
+
+[[ $CONFIG_SIGNALFILTERINGFRACTION -gt 100 ]] && CONFIG_SIGNALFILTERINGFRACTION="100"
+[[ $((CONFIG_PROCID%100)) -ge $CONFIG_SIGNALFILTERINGFRACTION ]] && CONFIG_SIGNALFILTERING="off" || CONFIG_SIGNALFILTERING="on"
```

Comment:
Ciao Michael,
Sorry, maybe i don't remember, but why do you do the check:
$((CONFIG_PROCID%100)) -ge $CONFIG_SIGNALFILTERINGFRACTION
?
Chiara

Reply:
Ah, ok, now I see :) Thanks!